lib/linter.js

```diff
@@ -1181,6 +1210,11 @@ class Linter {
      * @param {boolean} options.allowInlineConfig Flag indicating if inline comments
      *      should be allowed.
      * @param {boolean|Function} options.fix Determines whether fixes should be applied
+     * @param {Function} options.preprocess preprocessor for source text. If provided, this should
+     *      this should accept a string of source text, and return an array of code blocks to lint.
```

Comment:
"this should" is duplicated at the end of the previous line and the beginning of this line.